default/environment-py37.yml

```diff
@@ -1,25 +1,7 @@
 channels:
+  - coiled
   - conda-forge
-  - defaults
 dependencies:
   - python=3.7
-  - pip
-  - python-blosc
-  - cytoolz
-  - dask=2022.1.1  # Versions after this are known to be unstable
-  - lz4
-  - numpy>=1.19.0
-  - pandas>=1.3.0
-  - bokeh>=2.1.1
-  - dask-image>=0.3.0
-  - dask-ml>=1.5.0
-  - s3fs>=2021.8.0
-  - numba
-  - pyarrow>=0.15.1
-  - python-graphviz
-  - scikit-learn>=0.23.1
-  - h5py
-  - xarray
-  - bottleneck
-  - requests
-  - pillow>=7.2.0
+  - coiled-runtime=0.0.3
+  - coiled=0.0.73
```

Comment:
Pinning to the latest `coiled` release explicitly to work around https://github.com/coiled/coiled-runtime/issues/22

Reply:
the software environment doesn't even care about the `coiled` package, does it?

I'm skeptical of this pin...

Reply:
Yup. It's not used. Happy to remove it.

Reply:
@jrbourbeau I'd approve this MR if you remove the coiled pin.

I know that means some solves will can an old coiled version, but that's fine, software environments don't care what coiled version we use.

Reply:
oh Samantha makes a good point, let's check our examples

Reply:
Once https://github.com/coiled/coiled-runtime/pull/57 is in and released we can remove the `coiled` pin and users will get the latest `coiled` by default